lib/portlayer/util/util.go

```diff
@@ -53,3 +57,15 @@ func ServiceURL(serviceName string) *url.URL {
 
 	return s
 }
+
+// Update the VM display name on vSphere UI
+func DisplayName(id, name string) string {
+	shortID := id[:shortIDLen]
+	nameMaxLen := maxVMNameLength - len(shortID)
+	prettyName := name
```

Comment:
I think we can stick to the current approach now. In the future, maybe we could allow the customer himself to decide his container naming convention, as long as the name is within maxVMnameLength.